.github/workflows/build-hotfix.yml

```diff
@@ -65,3 +66,20 @@ jobs:
           git commit -m "chore(dependencies): Autobump oss clouddriver version"
           git pull
           git push
+      # Adding OSS docker build for SNAP
+      - name: Publish Release
+        if: startsWith(github.ref, 'refs/heads/release-')
+        run: |
+          ./gradlew final test docker dockerPush $GRADLE_ARGS
+          echo "BUILD_NUMBER=$(./gradlew -Prelease.useLastTag=true $GRADLE_ARGS  printVersion|grep VERSION|awk -F: '{print $2}')" >> $GITHUB_ENV
```

Comment:
This is also only applicable to armory extension projects like `clouddriver-armory`. The version number was already calculated in the step `build-info`, so that should be used for all `./gradlew` commands (`./gradlew -Pversion=${{ steps.build-info.outputs.version }} ...`).

Reply:
Using ${{ steps.build-info.outputs.version }} to set the docker image version.